src/components/Edge.vue

```diff
@@ -87,6 +62,40 @@ export default {
     padding () {
       return this.bezierOffset
     }
+  },
+  methods: {
+    getCenterStart () {
+      if (!this.isMounted) {
+        return { x: 0, y: 0 }
+      }
+      let x = this.start.x
+      let y = this.start.y
+      if (this.start instanceof HTMLElement) {
+        let rect = this.start.getBoundingClientRect()
+        let parentRect = this.$el.parentElement.getBoundingClientRect()
+        x = rect.x + rect.width / 2 - parentRect.x
+        y = rect.y + rect.height / 2 - parentRect.y
+      }
+      return { x, y }
+    },
+    getCenterEnd () {
+      if (!this.isMounted) {
+        return { x: 0, y: 0 }
+      }
+      let x = this.end.x
+      let y = this.end.y
+      if (this.end instanceof HTMLElement) {
+        let rect = this.end.getBoundingClientRect()
+        let parentRect = this.$el.parentElement.getBoundingClientRect()
+        x = rect.x + rect.width / 2 - parentRect.x
+        y = rect.y + rect.height / 2 - parentRect.y
+      }
+      return { x, y }
+    },
+    refresh () {
```

Comment:
Why not use `this.$forceUpdate()` instead?

Reply:
It seems that $forceUpdate will not recompute computed values and methods. A custom refresh method allows for more control and optimization. However, I find this unpretty myself.
Might there be an observable way to determine the bounding box or just the center point as an object field?